dateparser/timezones.py

```diff
@@ -1,393 +1,448 @@
-    ('NZDT', 46800),
-    ('NZMT', 41400),
-    ('NZST', 43200),
-    ('OMSST', 25200),
-    ('OMST', 21600),
-    ('ORAST', 18000),
-    ('ORAT', 18000),
-    ('PDDT', -21600),
-    ('PDT', -25200),
-    ('PEST', -14400),
-    ('PET', -18000),
-    ('PETST', 43200),
-    ('PETT', 43200),
-    ('PGT', 36000),
-    ('PHOT', 46800),
-    ('PHST', 32400),
-    ('PHT', 28800),
-    ('PKST', 21600),
-    ('PKT', 18000),
-    ('PLMT', 25620),
-    ('PMDT', -7200),
-    ('PMMT', 35340),
-    ('PMST', -10800),
-    ('PMT', 540),
-    ('PNT', -30600),
-    ('PONT', 39600),
-    ('PPMT', -17340),
-    ('PPT', -25200),
-    ('PST', -28800),
-    ('PT', -28800),
-    ('PWT', -25200),
-    ('PYST', -10800),
-    ('PYT', -14400),
-    ('QMT', -18840),
-    ('QYZST', 25200),
-    ('QYZT', 21600),
-    ('RET', 14400),
-    ('RMT', 3000),
-    ('ROTT', -10800),
-    ('SAKST', 43200),
-    ('SAKT', 39600),
-    ('SAMT', 14400),
-    ('SAST', 7200),
-    ('SBT', 39600),
-    ('SCT', 14400),
-    ('SDMT', -16800),
-    ('SDT', -36000),
-    ('SET', 3600),
-    ('SGT', 28800),
-    ('SHEST', 21600),
-    ('SHET', 18000),
-    ('SJMT', -20160),
-    ('SLT', 19800),
-    ('SMT', -13860),
-    ('SRET', 39600),
-    ('SRT', -10800),
-    ('SST', -39600),
-    ('STAT', 10800),
-    ('SVEST', 21600),
-    ('SVET', 14400),
-    ('SWAT', 5400),
-    ('SYOT', 10800),
-    ('TAHT', -36000),
-    ('TASST', 25200),
-    ('TAST', 21600),
-    ('TBIST', 18000),
-    ('TBIT', 10800),
-    ('TBMT', 10740),
-    ('TFT', 18000),
-    ('THA', 25200),
-    ('TJT', 18000),
-    ('TKT', -39600),
-    ('TLT', 32400),
-    ('TMT', 18000),
-    ('TOST', 50400),
-    ('TOT', 46800),
-    ('TRST', 14400),
-    ('TRT', 10800),
-    ('TSAT', 10800),
-    ('TVT', 43200),
-    ('ULAST', 32400),
-    ('ULAT', 28800),
-    ('URAST', 18000),
-    ('URAT', 18000),
-    ('UTC', 0),
-    ('UYHST', -9000),
-    ('UYST', -7200),
-    ('UYT', -10800),
-    ('UZST', 21600),
-    ('UZT', 18000),
-    ('VET', -16200),
-    ('VLAST', 39600),
-    ('VLAT', 36000),
-    ('VOLST', 14400),
-    ('VOLT', 14400),
-    ('VOST', 21600),
-    ('VUST', 43200),
-    ('VUT', 39600),
-    ('WARST', -10800),
-    ('WART', -14400),
-    ('WAST', 7200),
-    ('WAT', 3600),
-    ('WDT', 32400),
-    ('WEDT', 3600),
-    ('WEMT', 7200),
-    ('WEST', 3600),
-    ('WET', 0),
-    ('WFT', 43200),
-    ('WGST', -7200),
-    ('WGT', -10800),
-    ('WIB', 25200),
-    ('WIT', 32400),
-    ('WITA', 28800),
-    ('WMT', 5040),
-    ('WSDT', 50400),
-    ('WSST', 46800),
-    ('WST', 28800),
-    ('WT', 0),
-    ('XJT', 21600),
-    ('YAKST', 36000),
-    ('YAKT', 32400),
-    ('YAPT', 36000),
-    ('YDDT', -25200),
-    ('YDT', -28800),
-    ('YEKST', 21600),
-    ('YEKST', 21600),
-    ('YEKT', 18000),
-    ('YEKT', 18000),
-    ('YERST', 14400),
-    ('YERT', 10800),
-    ('YPT', -28800),
-    ('YST', -32400),
-    ('YWT', -28800),
-    ('zzz', 0)
+    {
+        'regex_patterns':
+            [r'(.)\(?%s\)?$',],
```

Comment:
I agree that using `.` is somewhat greedy. But is the date string in question representing a valid date? `MUTC` is not a valid timezone abbreviation, hence, should not be cleaned out of the string. What do you think?